internal/router/middleware.go

```diff
@@ -91,8 +94,32 @@ func JWTValidation(c context.Context, input *openapi3filter.AuthenticationInput)
 		if !ok {
 			return echo.ErrUnauthorized
 		}
-		ctx.Set(APIKey, true)
+		ctx.Set(apiKey, true)
 		return nil
 	}
 	return echo.ErrUnauthorized
 }
+
+// ProcessCLIFlags process handler flags
```

Comment:
what are the flags that this handler can process.
Its not clear to me what this is doing.

Reply:
timeout and status codes

Reply:
Thanks.  
Is the handlers a slice of strings that containing the URL path that maps to a handler?   i.e. ```/stream, /feature-configs```? 
And so here for each handler we can return a specified timeout or status code?  Then on line 119 is that a way of returning a global status if no handlers are specified ?

Reply:
it is actually operationId in swagger file
-s=401 -o=Authenticate

Reply:
but you can put just -s=401